docs/reference/project-reference.md

```diff
@@ -1,3 +1,179 @@
 # Project
 
-::: up42.project.Project
+The Project class enables access to the UP42 [analytics functionality](analytics.md). A project stores workflows and their corresponding job runs.
+
+```python
+project = up42.initialize_project()
+```
+
+## Projects
+
+### info
+
+The `info` attribute returns metadata of a specific UP42 project.
+
+The returned format is `dict`.
+
+<h5> Example </h5>
+
+```python
+project.info
+```
+
+### max_concurrent_jobs
+
+The `max_concurrent_jobs` attribute returns the maximum number of jobs that can be running simultaneously.
+
+The returned format is `int`.
+
+<h5> Example </h5>
+
+```python
+project.max_concurrent_jobs
+```
+
+### get_project_settings()
+
+The `get_project_settings()` function returns threshold limits applied to the project.
+
+```python
+get_project_settings()
+```
+The returned format is `List[Dict[str, str]]`.
```

Comment:
```suggestion
The returned format is `list[dict[str, str]]`.
```
nitpick: Python versions >= 3.9 support this. Since we using smaller case notations at other places, consider sticking to it.